kong/db/schema/entities/upstreams.lua

```diff
@@ -186,6 +190,10 @@ local r =  {
     { hash_fallback_header = typedefs.header_name, },
     { hash_on_cookie = { type = "string",  custom_validator = utils.validate_cookie_name }, },
     { hash_on_cookie_path = typedefs.path{ default = "/", }, },
+    { hash_on_query_arg = query_arg },
+    { hash_fallback_query_arg = query_arg },
+    { hash_on_uri_capture = { type = "string", len_min = 1 }, },
+    { hash_fallback_uri_capture = { type = "string", len_min = 1 }, },
```

Comment:
can we rename `query_arg` to `simple_param` (or similar) and reuse that typedef also for the 2 new uri fields? constraints are the same but query_arg gets a typedef, whilst the others aredefined in line